src/pykeen/datasets/base.py

```diff
@@ -651,21 +651,37 @@ def __init__(
     def _get_path(self) -> str:
         return os.path.join(self.cache_root, self.name)
 
+    def _extract(
```

Comment:
This change in the way the superclass works is incompatible with the DRKG loader, which was inheriting from this. I don't really see the need for extracting from the compressed file. Why not just load the things inside it dynamically?

Reply:
The Tarfile one should do the same than before (if I did not do a mistake there). Besides the Google Drive issues, it looked like the Zipfile one could be the same except this one line.

Reply:
yes I think you made a mistake. the point of this class was explicitly not to call the extract function on the entire archive, but rather you pass the path inside the tar file so the class knows how to open the file without extracting everything, which is then read into a pandas dataframe